src/workerd/api/hibernatable-web-socket.c++

```diff
@@ -155,14 +159,17 @@ kj::Promise<WorkerInterface::CustomEvent::Result>
       KJ_UNREACHABLE;
     }
     message.setWebsocketId(kj::mv(eventParameters.websocketId));
+    KJ_IF_SOME(t, eventParameters.eventTimeoutMs) {
+      message.setEventTimeoutMs(t);
+    }
   }
 
   return req.send().then([](auto resp) {
     auto respResult = resp.getResult();
     return WorkerInterface::CustomEvent::Result {
       .outcome = respResult.getOutcome(),
     };
-  });
+  }).eagerlyEvaluate(nullptr);
```

Comment:
For my own education, why do we need the `eagerlyEvaluate` here?  Does no one wait on the result for `sendRpc`?

Reply:
Ah! This was a leftover from the old PR, We don't need it indeed.